tests/core/test_yaml_handler.py

```diff
@@ -0,0 +1,71 @@
+import os.path
+import sys
+from pathlib import Path
+from typing import AnyStr, Optional
+
+import pytest
+from _pytest.capture import CaptureResult
```

Comment:
Since this is the internal API, what are the chances this breaks? I love the diligence around type annotations but I'd probably just slap an `Any` on this and not feel too bad.

Reply:
works for me :)  That's a good point about it being part of an internal API